doc/nrf/gs_installing.rst

```diff
@@ -209,6 +213,9 @@ To install west, reopen the command prompt window to ensure that Python is initi
 
          pip3 install west
 
+      .. note::
+         Ensure the west location is added to path in environmental variables.
```

Comment:
Shouldn't this note be added for linux and macOS also?
```suggestion
         Ensure the west location is added to the path in environmental variables.
```

Reply:
@FrancescoSer we discussed it for Windows specifically, not really sure how it works for Ubuntu and MacOS. Should we add this information to other tabs as well?

Reply:
This sentence is windows-only. Not sure how this is really an issue on Linux and MacOS